strings/tips.txt

```diff
@@ -112,11 +138,13 @@ As the Head of Personnel, you are not higher ranking than other heads of staff,
 As the Head of Personnel, you are just as large a target as the Captain because of the potential power your ID and computer can hand out.
 As the Mime, your invisible wall power blocks people as well as projectiles. You can use it in a pinch to delay your pursuer.
 As the Mime, you can use :r and :l to speak through your ventriloquist dummy.
-As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor.
+As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor, and might even make you valid, depending on the circumstance. A freshly-cloned Mime is not valid, as they lose their vow by default.
```

Comment:
"valid" stuff applies here, and frankly the freshly cloned stuff is so narrow that it makes for a shitty tip. That sounds more like behavior we should fix rather than warn people about tbh